src/EfficientDynamoDb/Configs/RegionEndpoint.cs

```diff
@@ -10,136 +10,141 @@ public class RegionEndpoint
         internal string RequestUri { get; }
         
         public string Region { get; }
+
+        static RegionEndpoint Create(string region, string uriFormat)
```

Comment:
Could you explicitly specify the access modifier here? We prefer to have them everywhere even if they match the default one.